TeensyDmx.h

```diff
@@ -34,6 +34,62 @@
 #define UART_C3_FEIE    (uint8_t)0x02   // Framing Error Interrupt Enable
 #endif
 
+// ----- macros -----
+
+// 16-bit and 32-bit integers in the RDM protocol are transmitted highbyte - lowbyte.
+// but the ATMEGA processors store them in highbyte - lowbyte order.
+// Use SWAPINT to swap the 2 bytes of an 16-bit int to match the byte order on the DMX Protocol.
+// avoid using this macro on variables but use it on the constant definitions.
+#define SWAPINT(i) (((i&0x00FF)<<8) | ((i&0xFF00)>>8))
+// Use SWAPINT32 to swap the 4 bytes of a 32-bit int to match the byte order on the DMX Protocol.
+#define SWAPINT32(i) ((i&0x000000ff)<<24) | ((i&0x0000ff00)<<8) | ((i&0x00ff0000)>>8) | ((i&0xff000000)>>24)
+
+// read a 16 bit number from a data buffer location
+#define READINT(p) ((p[0]<<8) | (p[1]))
+
+enum { DMX_BUFFER_SIZE = 512 };
+enum { RDM_BUFFER_SIZE = (24+231) }; //TODO(Peter): Check me
```

Comment:
What in particular needs checking?

Reply:
I just need to check the standard and confirm that really is the max buffer size.

Reply:
Since it's accessed as part of the union, might be better to define the RDM_BUFFER_SIZE as sizeof(struct RDMDATA)?

Reply:
Done in a future commit.